katas/content/complex_arithmetic/index.md

```diff
@@ -234,35 +233,33 @@ Another way to think about this is that you're taking a point that is $1$ unit a
 A complex number of the format $r \cdot e^{i\theta}$ will be represented by a point which is $r$ units away from the origin, in the direction specified by the angle $\theta$.
 Sometimes $\theta$ will be referred to as the number's **argument** or **phase**.
 
-> In Q#, complex numbers in polar form are represented as user-defined type `ComplexPolar` from the `Microsoft.Quantum.Math` namespace. 
-> 
-> You can convert a complex number $x = r \cdot e^{i\theta}$ into a tuple of two `Double` numbers using unwrap operator and tuple deconstruction: `let (r, theta) = x!;`, 
-> or access its magnitude and phase using their names: `let (r, theta) = (x::Magnitude, x::Argument);`.
-> 
-> You can construct a complex number from its magnitude and phase as follows: `let x = ComplexPolar(r, theta);`.
+> In Q#, complex numbers in polar form are represented as user-defined struct type `ComplexPolar` from the `Microsoft.Quantum.Math` namespace.
+>
+> Given a complex number $x = r \cdot e^{i\theta}$, you can access its magnitude and phase using their names: `let (r, theta) = (x.Magnitude, x.Argument);`.
+>
+> You can construct a complex number from its magnitude and phase as follows: `let x = new ComplexPolar{ Magnitude = r, Argument = theta };`.
```

Comment:
Is this the preferred way? Other places use ComplexPolar(a, b).

Reply:
I've updated this to use the older syntax for now, since it is more concise.